bin/evm2wasm.js

```diff
@@ -2,16 +2,98 @@
 
 const evm2wasm = require('../index.js')
 const ethUtil = require('ethereumjs-util')
+const argv = require('minimist')(process.argv.slice(2))
+const fs = require('fs')
 
-const input = ethUtil.toBuffer(process.argv[2])
-
-evm2wasm.evm2wasm(input, {
-  stackTrace: process.argv[3] === 'trace',
-  tempName: 'temp',
-  inlineOps: true,
-  wabt: true
-}).then(function (output) {
-  console.log(output.toString('binary'))
-}).catch(function (err) {
-  console.error('Failed: ' + err)
-})
+//read EVM bytecode from a file
+function readEVM (file) {
+  return new Promise((resolve, reject) => {
+    if (file) {
+      fs.readFile(file, (err, data) => {
```

Comment:
The same, could use `util.promisify(fs.readFile)`

Reply:
or fs.readFileSync